requirements.txt

```diff
@@ -2,13 +2,13 @@
 # This file is autogenerated by pip-compile with Python 3.10
 # by the following command:
 #
-#    pip-compile --extra=aafiguresupport --extra=mathsupport --extra=plantumlsupport --extra=rawhtmlsupport --extra=sphinx --extra=svgsupport --extra=tests --output-file=requirements.txt setup.py
+#    pip-compile --extra=aafiguresupport --extra=mathsupport --extra=plantumlsupport --extra=rawhtmlsupport --extra=sphinx --extra=svgsupport --extra=tests --output-file=requirements1.txt setup.py
```

Comment:
Why the changed filename?

Reply:
Because I failed to read the instructions properly when creating it...

Reply:
> Why are we skipping sphinx this time? A non-scientific tests run with sphinx 6.1.3 and python 3.10 completed successfully for me

I saw it was a major version upgrade and assumed we'd break as we did with 4 to 5 without checking. As we don't, I've bumped this too.